packages/babel-plugin-transform-modules-systemjs/src/index.ts

```diff
@@ -190,7 +190,7 @@ export default declare<PluginState>((api, options: Options) => {
             return;
           }
           const exportedNames = this.exports[name];
-          if (!exportedNames) return;
+          if (!exportedNames) continue;
           for (const exportedName of exportedNames) {
```

Comment:
If here find not matched export name, here will go out of function. The name inside of export will not match.